README.md

```diff
@@ -65,15 +74,21 @@ There are community maintained instructions available here:
 [Running Leela Zero client on a Tesla V100 GPU for free (Google Cloud Free Trial, Microsoft Azure, Oracle cloud, etc)](https://docs.google.com/document/d/1P_c-RbeLKjv1umc4rMEgvIVrUUZSeY0WAtYHjaxjD64/edit?usp=sharing)
 
 
-# I just want to play right now
+# I want to run leela-zero now (main instructions)
```

Comment:
The instructions for someone who just wants to play should not be totally hidden away on the bottom.